.travis.yml

```diff
@@ -16,49 +16,51 @@ branches:
   only:
   - master
 
-jobs:
-  include:
-    - env: "MODE=lint"
-    - env: "MODE=aot"
-    - env: "MODE=hello-world"
-    - env: "MODE=prerender"
-    - env: "MODE=ssr"
-    - env: "MODE=saucelabs_required"
-    - env: "MODE=browserstack_required"
-    - env: "MODE=travis_required"
-    - stage: Deploy
-      env: "DEPLOY_MODE=build-artifacts"
-      if: type = push
+#jobs:
+#  include:
+#    - env: "MODE=lint"
+#    - env: "MODE=aot"
+#    - env: "MODE=hello-world"
+#    - env: "MODE=prerender"
+#    - env: "MODE=ssr"
+#    - env: "MODE=saucelabs_required"
+#    - env: "MODE=browserstack_required"
+#    - env: "MODE=travis_required"
+#    - stage: Deploy
+#      env: "DEPLOY_MODE=build-artifacts"
+#      if: type = push
     # Closure Compiler CI check is temporarily disabled until a new version of
     # the tool is released with https://github.com/google/closure-compiler/pull/2600
     # - env: "MODE=closure-compiler"
-env:
-  global:
-  - LOGS_DIR=/tmp/flex-layout-build/logs
-  - SAUCE_USERNAME=angular-ci
-  - BROWSER_STACK_USERNAME=adamplumer1
-  - BROWSER_STACK_ACCESS_KEY=WgLjxoB2zQ3tqmsznKnz
-  - BROWSER_PROVIDER_READY_FILE=/tmp/flex-layout-build/readyfile
-  - BROWSER_PROVIDER_ERROR_FILE=/tmp/flex-layout-build/errorfile
+#env:
+#  global:
+#  - LOGS_DIR=/tmp/flex-layout-build/logs
+#  - SAUCE_USERNAME=angular-ci
+#  - BROWSER_STACK_USERNAME=adamplumer1
+#  - BROWSER_STACK_ACCESS_KEY=WgLjxoB2zQ3tqmsznKnz
+#  - BROWSER_PROVIDER_READY_FILE=/tmp/flex-layout-build/readyfile
+#  - BROWSER_PROVIDER_ERROR_FILE=/tmp/flex-layout-build/errorfile
 
-matrix:
-  allow_failures:
-    - env: "MODE=ssr"
+#matrix:
+#  allow_failures:
+#    - env: "MODE=ssr"
 
 
-before_install:
-  - source ./scripts/ci/env.sh
+#before_install:
+#  - source ./scripts/ci/env.sh
+#
+#install:
+#  - yarn install --frozen-lockfile --non-interactive
+#
+#before_script:
+#  - mkdir -p $LOGS_DIR
 
-install:
-  - yarn install --frozen-lockfile --non-interactive
+#script:
+  # - ./scripts/ci/travis-script.sh
 
-before_script:
-  - mkdir -p $LOGS_DIR
+script: exit 0
 
-script:
-  - ./scripts/ci/travis-script.sh
-
-cache:
-  yarn: true
-  directories:
-    - ./node_modules/
+#cache:
+#  yarn: true
+#  directories:
+#    - ./node_modules/
```

Comment:
If this file is mostly just commented code, just remove it. Travis will be gone once that PR has been merged.

Reply:
Travis is still required, so if this gets merged before that is turned off, it still needs to run. If Jeremy turns off Travis pre-merge I'll delete the file, ow will remove in a follow-up